cylc/flow/ws_messages.proto

```diff
@@ -231,3 +234,51 @@ message PbEntireWorkflow {
     repeated PbFamilyProxy family_proxies = 6;
     repeated PbEdge edges = 7;
 }
+
+message EDeltas {
+    double time = 1;
+    int64 checksum = 2;
+    repeated string pruned = 3;
+    repeated PbEdge deltas = 4;
+    bool reloaded = 5;
```

Comment:
Can I just check what the `reloaded` field means?

Am I right in thinking that it is `False` if the object hasn't been sent before (i.e. it is a new edge) and `True` if it is a delta representing a change to the original?

Reply:
Each iteration is a new set of deltas, `reloaded` is to indicate that the workflow has reloaded and the delta is a full payload, so elements therein should replace (not merge with) the respective data-store topic (i.e. new set of edges in `deltas` to replace old `data[‘edges’]`)

Reply:
So if `reloaded` is `True` then any fields which aren't present in the delta should be removed from the sync'ed object?

Reply:
Exactly.. I use a dict comprehension on the deltas to replace data store topic i.e. 
`self.data[w_id][topic] = {ele.id: ele for ele in delta.deltas}`
https://github.com/dwsutherland/cylc-uiserver/blob/71288f0febe03089c4f87755861029be8808e0f3/cylc/uiserver/data_mgr.py#L141

Reply:
Yep the UI Server code makes its purpose clearer, thanks!